package.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
I approved although I was not considering this plugin to have any JS assets. I'd prefer a base for such a thing that could be included when the project needs it. Although this file here is just a "formality" to make it a npm package.

Reply:
I understand but I prefer to make it a "statement" in the present moment. Besides, it does nothing and doesn't force anything.

> Although this file here is just a "formality" to make it a npm package.

Not only for that.